group/assignment5/Testing.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <algorithm>
+#include <vector>
+#include <stack>
+#include <map>
+#include <unordered_map>
+#include <unordered_set>
+#include "TestingMacros.h"
+
+void givenTest() {
+  AlphabetFinder AF({"ART", "RAT", "CAT", "CAR"});

```

Comment:
Just a side note that if we decide to call the dictionary by const reference, then using temporary vectors as in `AlphabetFinder AF({"ART", "RAT", "CAT", "CAR"});` will cause an error and we'll need to do something like this: 
```
vector<string> dictionary {"ART", "RAT", "CAT", "CAR"};
AlphabetFinder AF(dictionary);
```